src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -54,15 +54,14 @@ public void storeMessage(Message message) {
    *     message. List is sorted by time descending.
    */
   public List<Message> getMessages(String user) {
-  	
-    Query query =
-        new Query("Message")
-            .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
-            .addSort("timestamp", SortDirection.DESCENDING);
-    PreparedQuery results = datastore.prepare(query);
-
-   return convertEntitiesToMessages(results);
-  }
+	  	
```

Comment:
It looks like this doesn't actually change any functionality and just modifies the whitespace. Please revert this back to the old version so the whitespace matches the rest of the file. I think you may need to update your editor to use spaces instead of tabs to avoid this in the future.